src/Files.App/Filesystem/FilesystemOperations/Helpers/FilesystemHelpers.cs

```diff
@@ -778,13 +783,76 @@ public static async Task<IEnumerable<IStorageItemWithPath>> GetDraggedStorageIte
 					return itemsList;
 				}
 			}
-			if (packageView.Properties.TryGetValue("FileDrop", out var data))
```

Comment:
Pretty cool. I don't have access to a PC but could you check if we set the "FileDrop" field of packageView.Properties somewhere (if you have not checked already)? Code above might still be needed, this "FileDrop" is different from the "FileDrop" clipboard format, it's (was?) something we set from inside Files.

Reply:
yeah, I noticed that. there's no other references to it in the codebase, I believe #9014 removed what you're referring to